.docs/user-guide/demo.md

```diff
@@ -0,0 +1,153 @@
+# Demo
+
+## REX-Ray with Two-Node libStorage Client/Server
+
+This demo consists of a two-node deployment with the first node configured as a REX-Ray/libStorage server and the second node as merely a client. Both nodes have Docker installed and configured to leverage REX-Ray for persistent storage.
```

Comment:
Hi @kacole2,

I've previously noted for sake of readability and long-term support it's a documented request to limit Markdown to a width of 80 columns. Please reformat the above text. Thanks!

I'm not going to keep attaching this comment, you get the idea :)